.github/workflows/release-toolbox.yml

```diff
@@ -0,0 +1,204 @@
+        with:
+          args: --acl public-read --follow-symlinks
+        env:
+          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: "eu-west-2"
+          SOURCE_DIR: "packages/graphql-toolbox/dist" # optional: defaults to entire repository
+
+  merge-tag-push:
+    needs:
+      - bump-version
+      - deploy
+
+    runs-on: ubuntu-latest
+
+    environment:
+      name: github
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: master
+          fetch-depth: 0
+          token: ${{ secrets.NEO4J_TEAM_GRAPHQL_PERSONAL_ACCESS_TOKEN }}
+      - name: git config
+        run: |
+          git config --global user.name 'Neo4j Team GraphQL'
+          git config --global user.email 'team-graphql@neotechnology.com'
+      - name: git merge
+        run: git merge --no-ff origin/release/graphql-toolbox/${{ needs.bump-version.outputs.version }} --strategy-option theirs
+      - name: git tag
+        run: |
+          git tag -a @neo4j/graphql-toolbox@${{ needs.bump-version.outputs.version }} -m "@neo4j/graphql-toolbox@${{ needs.bump-version.outputs.version }}"
+      - name: git push
+        run: git push --follow-tags
+      - uses: actions/checkout@v3
+        with:
+          ref: dev
+          fetch-depth: 0
+          token: ${{ secrets.NEO4J_TEAM_GRAPHQL_PERSONAL_ACCESS_TOKEN }}
+      - name: git merge
+        run: git merge --no-ff origin/release/graphql-toolbox/${{ needs.bump-version.outputs.version }} --strategy-option theirs
+      - name: git push
+        run: git push
+
+  delete-release-branch:
+    needs:
+      - bump-version
+      - merge-tag-push
+
+    runs-on: ubuntu-latest
+
+    environment:
+      name: github
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.NEO4J_TEAM_GRAPHQL_PERSONAL_ACCESS_TOKEN }}
+      - name: Delete release branch
+        run: |
+          git config --global user.name 'Neo4j Team GraphQL'
+          git config --global user.email 'team-graphql@neotechnology.com'
+          git push origin --delete release/graphql-toolbox/${{ needs.bump-version.outputs.version }}
+
+  changelog:
+    needs:
+      - bump-version
+      - merge-tag-push
+
+    strategy:
+      matrix:
+        package:
+          - graphql-toolbox
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Extract prerelease from version
+        id: get-prerelease
+        run: |
+          prerelease=$(echo ${{ needs.bump-version.outputs.version }} | cut -d@ -f3 | cut -d- -f2)
+          echo "::set-output name=prerelease::$prerelease"
+      - name: Build Changelog
+        id: github-release
+        uses: mikepenz/release-changelog-builder-action@v3
+        with:
+          configuration: "config/release-changelog-builder-action/configuration-graphql-toolbox.json"
```

Comment:
This file will need adding, can be copied and slightly tweaked from one of the others. 🙂

Reply:
Thanks for pointing it out. I will do that!